src/components/Modals/AddContactModal.jsx

```diff
@@ -151,39 +154,56 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
               )
             }}
           />
-          <DialogActions sx={{ width: '100%' }}>
-            <Box
-              sx={{
-                display: 'flex',
-                flexDirection: isSmallScreen ? 'column' : 'row',
-                gap: isSmallScreen ? '10px' : '8px',
-                justifyContent: 'center',
-                alignItems: 'center',
-                width: '100%'
-              }}
+          <InputLabel id="relationship-label">Relationship</InputLabel>
+          <Select
+            labelId="relationship-label"
+            id="relationship"
+            name="relationship"
+            value={relationship}
+            label="Relationship"
+            onChange={(e) => setRelationship(e.target.value)}
+            fullWidth
+          >
+            <MenuItem value={RELATIONSHIPS.Blank} />
+            <MenuItem value={RELATIONSHIPS.Client}>Client</MenuItem>
+            <MenuItem value={RELATIONSHIPS.CaseManagement}>Case Management</MenuItem>
+            <MenuItem value={RELATIONSHIPS.AssociatedOrg}>Associated Organization</MenuItem>
+          </Select>
+          <InputLabel id="relationship-status-label">Relationship Status</InputLabel>
+          <Select
+            labelId="relationship-status-label"
+            id="relationshipStatus"
+            name="relationshipStatus"
+            value={relationshipStatus}
+            label="RelationshipStatus"
+            onChange={(e) => setRelationshipStatus(e.target.value)}
+            fullWidth
+          >
+            <MenuItem value={RELATIONSHIP_STATUS.BLANK} />
+            <MenuItem value={RELATIONSHIP_STATUS.ACTIVE}>Active</MenuItem>
+            <MenuItem value={RELATIONSHIP_STATUS.ARCHIVED}>Archived</MenuItem>
+            <MenuItem value={RELATIONSHIP_STATUS.ETC}>etc</MenuItem>
+          </Select>
```

Comment:
Personally, I think we should revert the modal back to what we had before until we're sure where to include the status updates for "relationship" (or "relationship type", which I think is the better wording) and "relationship status".

Like what I've mentioned earlier, "relationship status" (active/archive/other) feels like it should belong to the contacts list rather than in the add contacts modal and "relationship type" feels like it should belong to the sign up page, although, having it in the contacts list would also work.

If we were to include these `<Select>` components in this PR, I think should have them placed there instead.